iqm5q.py

```diff
@@ -17,6 +17,7 @@
 )
 
 RUNCARD = pathlib.Path(__file__).parent / "iqm5q.yml"
+KERNEL_FOLDER = pathlib.Path(__file__).parent / "iqm5q_kernels/"
```

Comment:
Let's just use a single folder for all extra stuffs
```suggestion
FOLDER = pathlib.Path(__file__).parent / "iqm5q"
```